src/shim/README.md

```diff
@@ -140,16 +140,3 @@ The [`@dojo/framework/shim/string`](../../docs/shim/string.md) module contains `
 
 `@dojo/framework/shim/Symbol` provides an implementation of the [ES2015 Symbol specification](http://www.ecma-international.org/ecma-262/6.0/#sec-symbol-objects) for environments that do not natively support `Symbol`.
 
-<!-- doc-viewer-config
-{
-	"api": "docs/shim/api.json",
-	"pages": [
-		"docs/shim/array.md",
```

Comment:
I'm not sure if this is needed to render the menu links? core doesn't have a reference to `lang` in this form but the menu link seems to render with lang as a sub page.

Reply:
It used to, so that's why there is a rendered sub item: 

https://github.com/dojo/framework/commit/5b24ed21792709d44530a5f8c9c5d34d05e6a9b6#diff-4af3387b9fd86cec04b80c2b5bebfa1aL14